lang/src/org/partiql/lang/eval/physical/PhysicalBexprToThunkConverter.kt

```diff
@@ -291,6 +292,34 @@ internal class PhysicalBexprToThunkConverter(
         val value = exprConverter.convert(spec.expr).toValueExpr(spec.expr.metas.sourceLocationMeta)
         CompiledSortKey(comp, value)
     }
+
+    // TODO: Remove from experimental once https://github.com/partiql/partiql-docs/issues/31 is resolved and a RFC is approved
+    override fun convertWindow(node: PartiqlPhysical.Bexpr.Window): RelationThunkEnv {
+        val source = this.convert(node.source)
+
+        val windowPartitionList = node.windowSpecification.partitionBy
+
+        val windowSortSpecList = node.windowSpecification.orderBy
+
+        val compiledPartitionBy = windowPartitionList?.exprs?.map {
+            exprConverter.convert(it).toValueExpr(it.metas.sourceLocationMeta)
+        } ?: emptyList()
+
+        val compiledOrderBy = windowSortSpecList?.sortSpecs?.let { compileSortSpecs(it) } ?: emptyList()
+
+        val compiledWindowFunctionParameter = node.windowExpression.args.map {
+            exprConverter.convert(it).toValueExpr(it.metas.sourceLocationMeta)
+        }
+
+        // locate operator factory
+        val factory = findOperatorFactory<WindowRelationalOperatorFactory>(RelationalOperatorKind.WINDOW, node.i.name.text)
+
+        // create operator implementation
+        val bindingsExpr = factory.create(node.i, source, compiledPartitionBy, compiledOrderBy, node.windowExpression, compiledWindowFunctionParameter)
```

Comment:
Ideally we'd want some sort ofobject representing the compiled window expression / function -- but I know you're working on the representation of the function -- so I'll just wait for those changes in the next PR.